Description: Upstream changes introduced in version 1.4.1-0oneiric
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 ros-fuerte-octovis (1.4.1-0oneiric) oneiric; urgency=high
 .
   * autogenerated
 .
 The person named in the Author field signed this changelog entry.
Author: Armin Hornung <hornunga@informatik.uni-freiburg.de>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- ros-fuerte-octovis-1.4.1.orig/CMakeLists.txt
+++ ros-fuerte-octovis-1.4.1/CMakeLists.txt
@@ -40,6 +40,8 @@ find_package(octomap REQUIRED
 )
 INCLUDE_DIRECTORIES(${OCTOMAP_INCLUDE_DIRS})
 LINK_DIRECTORIES(${OCTOMAP_LIBRARY_DIRS})
+MESSAGE("Found OCTOMAP at " ${OCTOMAP_LIBRARY_DIRS} )
+
 
 # Export the package for use from the build-tree
 # (this registers the build-tree with a global CMake-registry)
